src/libawkward/array/IndexedArray.cpp

```diff
@@ -1880,8 +1880,8 @@ namespace awkward {
         Index64 outoffsets(starts.length() + 1);
         if (starts.length() > 0  &&  starts.getitem_at_nowrap(0) != 0) {
           throw std::runtime_error(
-            "reduce_next with unbranching depth > negaxis expects a "
-            "ListOffsetArray64 whose offsets start at zero");
+            std::string("reduce_next with unbranching depth > negaxis expects a "
+            "ListOffsetArray64 whose offsets start at zero ") + std::to_string(starts.ptr_lib()));
```

Comment:
This might have been debugging code. It wouldn't make sense as an error message. (The enum is just an integer, and the message says nothing about the fact that you're expecting a `ptr_lib` enum.)